README.md

```diff
@@ -5,6 +5,32 @@ HAML wrappers for [Twitter Bootstrap](http://getbootstrap.com/2.3.2) components.
 [![Build Status](https://travis-ci.org/lumoslabs/bootstrap_haml_helpers.svg)](https://travis-ci.org/lumoslabs/bootstrap_haml_helpers)
 [![Gem Version](https://badge.fury.io/rb/bootstrap_haml_helpers.svg)](http://badge.fury.io/rb/bootstrap_haml_helpers)
 
+How To Use The Components
+----
+```haml
+.container
+  .hero-unit
+    %h1 wow mom
+.container
+  .row
+    .span12
+      %h2 hello
+
+.container
+  .carousel.slide
+    .carousel-inner
+      .item.active
+        first item
+      .item
+        second item
```

Comment:
we prob don't need any of the stuff above here, it's not really using haml helpers...just the `= container do` and `=progress_bar do`